bin/wsl/diag.wsl2.sh

```
@@ -0,0 +1,391 @@
+}
+
+# Docker network info
+function show_docker_info() {
+    if command -v docker &>/dev/null; then
+        log "section" "Docker Network Information"
+        docker network ls
+
+        log "section" "Docker Containers with IPs"
+        containers=$(docker ps -q)
+        if [[ -n "${containers}" ]]; then
+            local output=$(docker ps -q | xargs -r docker inspect --format '{{.Name}} - {{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' | sed 's#^/##')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)